widgets/pagination.js

```diff
@@ -1,6 +1,8 @@
 var React = require('react');
 
 var utils = require('../lib/utils.js');
+var autoHide = require('../decorators/autoHide');
+var Pagination = autoHide(require('../components/Pagination/Pagination.js'));
```

Comment:
Do you mean smg like `PaginationAutoHide` and then `RefinementListHeaderFooterAutoHide`?